internal/controller/vrg_volrep.go

```diff
@@ -505,6 +549,8 @@ func (v *VRGInstance) preparePVCForVRDeletion(pvc *corev1.PersistentVolumeClaim,
 	delete(pvc.Annotations, pvcVRAnnotationProtectedKey)
 	delete(pvc.Annotations, pvcVRAnnotationArchivedKey)
 
+	delete(pvc.Labels, ConsistencyGroupLabel)
```

Comment:
As any change to the PVCs in the volume group requires a resync, what issue does this cause? If we remove the CG label per PVC, that would mean a group change, does that impact the overall workflow?